packaging/greenboot/microshift-running-check.sh

```diff
@@ -25,6 +25,36 @@ function forced_termination() {
     exit 1
 }
 
+# Check preconditions for existence of topolvm deployment.
+# Adapted from MicroShift code.
+#
+# args: None
+# return: 0 if topolvm readiness should be checked, 1 otherwise
+function topolvmShouldBeDeployed() {
+    if ! hash vgs 2>/dev/null; then
+        return 1
+    fi
+
+    # TODO: If lvmd.yaml exists, should we expect topolvm even if the OS doesn't has lvms2 installed?
+    if [ -f /etc/microshift/lvmd.yaml ]; then
+        return 0
+    fi
+
+    local -r volume_groups=$(vgs --readonly --options=name --noheadings)
+    if [ -z "${volume_groups}" ]; then
+        return 1
+    fi
+
+    local -r volume_groups_count=$(echo "${volume_groups}" | wc -l)
```

Comment:
I think we should count words to avoid formatting problems.
```suggestion
    local -r volume_groups_count=$(echo "${volume_groups}" | wc -w)
```

Reply:
Yes! `wc -l` was giving me 1 even on empty output, this helps, thx